redash/settings/__init__.py

```diff
@@ -138,6 +138,11 @@
 GOOGLE_CLIENT_SECRET = os.environ.get("REDASH_GOOGLE_CLIENT_SECRET", "")
 GOOGLE_OAUTH_ENABLED = bool(GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET)
 
+OIDC_CLIENT_ID = os.environ.get("REDASH_OIDC_CLIENT_ID", "")
+OIDC_CLIENT_SECRET = os.environ.get("REDASH_OIDC_CLIENT_SECRET", "")
+OIDC_COFIGURATION_URL = os.environ.get("REDASH_OIDC_COFIGURATION_URL", "")
```

Comment:
`CO(N)FIGURATION` spelled wrongly

Usually we see `*ISSUER_URL` or `*WELL_KNOWN_URL` as the env var names to the `$1/.well-known/openid-configuration` json to get the other endpoints.

Reply:
> CO(N)FIGURATION spelled wrongly

Oops.  We should probably try to fix that somehow (without causing issues for our users). :smile:

Reply:
Oops, let me fix that.
Also, I need some feedback on the overall approach. I am thinking of moving all OIDC logins through this code, including Google's. To keep things backward compatible, will keep Google's env variables and map them to new generic ones. The domain verification logic will only work for Google OIDC in the generic flow.
Thoughts?

Reply:
Oops! Let me fix that.